po/stellarium-sky/az.po

```diff
@@ -0,0 +1,10800 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
I know this is nitpickery, but were these comments not designed to provide some actual information? Looks unfinished IMO. (Given their particular nature, this "short" file is the only .po file I briefly inspected.)